v5_ws_public.go

```diff
@@ -71,6 +77,9 @@ const (
 
 	// V5WebsocketPublicTopicTicker :
 	V5WebsocketPublicTopicTicker = V5WebsocketPublicTopic("tickers")
+
+	// V5WebsocketPublicTopicTrade :
+	V5WebsocketPublicTopicTrade = V5WebsocketPublicTopic("publicTrade")
```

Comment:
Should this have just trade name instead of public trade ?

Reply:
No from the documentation, it's publicTrade
https://bybit-exchange.github.io/docs/v5/websocket/public/trade